src/access/universal.test.ts

```diff
@@ -1273,110 +1272,6 @@ describe("getGroupAccessAll", () => {
-    const options = {
-      fetch: jest.fn().mockResolvedValue({
-        ok: false,
-      } as never) as typeof fetch,
-    };
-    await getAccessFor(
-      "https://some.resource",
-      "agent",
-      "https://some.pod/profile#webid",
-      options
-    );
-    expect(universalModule.getAgentAccess).toHaveBeenCalledWith(
-      "https://some.resource",
-      "https://some.pod/profile#webid",
-      options
-    );
-  });
-
-  it("throws if the agent has been omitted", async () => {
-    const options = {
-      fetch: jest.fn() as typeof fetch,
-    };
-    await expect(
-      getAccessFor(
-        "https://some.resource",
-        ("agent" as unknown) as "public",
-        options
-      )
-    ).rejects.toThrow(
-      "When reading Agent-specific access, the given agent cannot be left undefined."
-    );
-  });
-
-  it("calls to getGroupAccess with the appropriate parameters", async () => {
-    const universalModule = jest.requireActual("./universal") as {
-      getGroupAccess: () => Promise<Access | null>;
-    };
-    universalModule.getGroupAccess = jest.fn();
-    const options = {
-      fetch: jest.fn() as typeof fetch,
-    };
-    await getAccessFor(
-      "https://some.resource",
-      "group",
-      "https://some.pod/groups#group",
-      options
-    );
-    expect(universalModule.getGroupAccess).toHaveBeenCalledWith(
-      "https://some.resource",
-      "https://some.pod/groups#group",
-      options
-    );
-  });
-
-  it("throws if the group has been omitted", async () => {
-    await expect(
-      getAccessFor("https://some.resource", ("group" as unknown) as "public")
-    ).rejects.toThrow(
-      "When reading Group-specific access, the given group cannot be left undefined."
-    );
-  });
-
-  it("throws if an actor is specified for public", async () => {
-    await expect(
-      getAccessFor(
-        "https://some.resource",
-        "public",
-        ("some actor" as unknown) as { fetch: typeof fetch }
-      )
-    ).rejects.toThrow(
-      "When reading public access, no actor type should be specified (here [some actor])."
-    );
-  });
-
-  it("calls to getPublicAccess with the appropriate parameters", async () => {
-    const universalModule = jest.requireActual("./universal") as {
-      getPublicAccess: () => Promise<Access | null>;
-    };
-    universalModule.getPublicAccess = jest.fn();
-
-    const options = {
-      fetch: jest.fn() as typeof fetch,
-    };
-    await getAccessFor("https://some.resource", "public", options);
-    expect(universalModule.getPublicAccess).toHaveBeenCalledWith(
-      "https://some.resource",
-      options
-    );
-  });
-
-  it("returns null if an unknown actor type is given", async () => {
-    await expect(
-      getAccessFor(
-        "https://some.resource",
-        ("unknown-actor" as unknown) as "public"
-      )
-    ).resolves.toBeNull();
-  });
-
   it("re-exports getAccessFrom", () => {
     expect(reexport_getAccessFor).toBeDefined();
```

Comment:
I suppose `getAccessForAll` should also be re-exported?

Reply:
eb26cabf165a204d7d57588ac2b8b4d9a1df8059